Mathlib/RingTheory/LittleWedderburn.lean

```diff
@@ -0,0 +1,162 @@
+/-
+Copyright (c) 2021 Johan Commelin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin, Eric Rodriguez
+-/
+import Mathlib.GroupTheory.ClassEquation
+import Mathlib.GroupTheory.GroupAction.ConjAct
+import Mathlib.RingTheory.Polynomial.Cyclotomic.Eval
+
+/-!
+# Wedderburn's Little Theorem
+
+This file proves Wedderburn's Little Theorem.
+
+## Main Declarations
+
+* `littleWedderburn`: a finite division ring is a field.
+
+## Future work
```

Comment:
Can you add here the results discussed on Zulip? There is also https://en.wikipedia.org/wiki/Artin%E2%80%93Zorn_theorem